docs/A-Introduction/information.md

```diff
@@ -8,115 +8,113 @@ sidebar_position: 2
+| 11111111             | <center>0xFF</center> | <center>255</center> |
 
-For example, the hexadecimal value 0x5C is equivalent to the 8-bit value 01011100<sub>2</sub>, which is equivalent to the decimal value 92. 
+For example, the hexadecimal value 0x5C is equivalent to the 8-bit value 01011100<sub>2</sub>, which is equivalent to the decimal value 92.
 
 To learn how to convert between hexadecimal and binary refer to the chapter entitled [Data Conversions](../Resources-Appendices/data-conversions.md) in the Appendices.
 
 ### Memory Model
 
-The memory model for organizing information stored in RAM is linear.  Any byte in memory is accessible through a map that treats each actual physical memory location as a position in a continuous sequence of locations aligned next to one another.
+The memory model for organizing information stored in RAM is linear. Any byte in memory is accessible through a map that treats each actual physical memory location as a position in a continuous sequence of locations aligned next to one another.
 
-![RAM memory model: linearly mapped \(continuous sequence\)](https://ict.senecacollege.ca//~ipc144/pages/images/ram.png)
+![RAM memory model: linearly mapped (continuous sequence)](https://ict.senecacollege.ca//~ipc144/pages/images/ram.png)
 
 ### Addresses
 
-Each byte of RAM has a unique address.  Addressing starts at zero, is sequential, and ends at the address equal to the size of RAM less 1 unit. 
+Each byte of RAM has a unique address. Addressing starts at zero, is sequential, and ends at the address equal to the size of RAM less 1 unit.
 
 For example, 4 Gigabytes of RAM
 
-* consists of 32 (= 4 * 8) Gigabits
-* starts at a low address of 0x00000000
-* ends at a high address of 0xFFFFFFFF
+- consists of 32 (= 4 \* 8) Gigabits
+- starts at a low address of 0x00000000
+- ends at a high address of 0xFFFFFFFF
 
 ![](/img/image21.png)
 
->**Note** that each byte, and not each bit, has its own address. We say that RAM is byte-addressable.
-
+> **Note** that each byte, and not each bit, has its own address. We say that RAM is byte-addressable.
 
 ### Sets of Bytes
 
 The abbreviations for sets of bytes are:
 
-* Kilo or k (=1024): 1 Kilobyte = 1024 bytes ~ 10<sup>3</sup> bytes
-* Mega or M (=1024k): 1 Megabyte = 1024 * 1024 bytes ~ 10<sup>6</sup> bytes
-* Giga or G (=1024M): 1 Gigabyte = 1024 * 1024 * 1024 bytes ~ 10<sup>9</sup> bytes
-* Tera or T (=1024G): 1 Terabyte = 1024 * 1024 * 1024 * 1024 bytes ~ 10<sup>12</sup> bytes
-* Peta or P (=1024T): 1 Petabyte = 1024 * 1024 * 1024 * 1024 * 1024 bytes ~ 10<sup>15</sup> bytes
-* Exa or E (=1024P): 1 Exabyte = 1024 * 1024 * 1024 * 1024 * 1024 * 1024 bytes ~ 10<sup>18</sup> bytes
+- Kilo or k (=1024): 1 Kilobyte = 1024 bytes ~ 10<sup>3</sup> bytes
+- Mega or M (=1024k): 1 Megabyte = 1024 \* 1024 bytes ~ 10<sup>6</sup> bytes
+- Giga or G (=1024M): 1 Gigabyte = 1024 _ 1024 _ 1024 bytes ~ 10<sup>9</sup> bytes
```

Comment:
But look at this change, it's changed `*` to `_` which is semantically not the same thing.  I don't understand it.  I wonder if we need to embed this code in backticks to have it not get changed?

Reply:
Somehow Prettier thought the `*` in `1024 * 1024 * 1024` as an indication for bulleted list or something and changed it into `_`. I can revert the formatting commit and embed this code in backticks to see if it helps.

Reply:
I noticed that some lines used `\` to escape the `*` and some don't. I think that's why Prettier decided to change `1024 * 1024 * 1024` into `1024 _ 1024 _ 1024`.

Reply:
I just reverted the formatting commit. I think we can either embed the line of code in backticks or escape it with `\`